lms/envs/common.py

```diff
@@ -1860,6 +1860,9 @@
     # Microsite configuration
     'microsite_configuration',
 
+    # RSS Proxy
+    'rss_proxy',
```

Comment:
Fair enough -- I'm not inclined to pursue it any further :) Only other thing I'll throw out there is whether or not this new djangoapp should be located in the "openedx" directory -- it seems like many new apps are being added there vs. lms.  If lms is the right place, that's fine.